xcp_d/data/reports.yml

```diff
@@ -16,3 +16,41 @@ sections:
 - name: About
   reportlets:
   - bids: {datatype: figures, desc: about, suffix: bold}
+# - name: Executive Summary
```

Comment:
Do you want to keep these changes in?

Reply:
I think so, in case we want to build on them later - what do you think?

Reply:
Fair enough. Happy to keep them in.